### PR TITLE
Weak FunExt implies FunExt

### PR DESCRIPTION
**Pull request by thchatzidiamantis**
Converse direction for #73. Also added a naive-funext axiom.

### CODE REVIEW

File: src/hott/08-families-of-maps.rzk.md [hunk 1]
```diff
@@ -694,6 +694,136 @@ contractible.
...
+  ( weakfunext : WeakFunExt)
+  : FunExt
+  :=
+    funext-weirdfunext (weirdfunext-weakfunext weakfunext)
+```
+
+The proof of `weakfunext-funext` from `06-contractible.rzk` works with a weaker
+version of function extensionality only requiring the map in the converse
+direction. We can then prove a cycle of implications between FunExt, NaiveFunExt
+and WeakFunExt.
```
Comment (fizruk) [inline src/hott/08-families-of-maps.rzk.md:793]:
```suggestion
direction. We can then prove a cycle of implications between `#!rzk FunExt`, `#!rzk NaiveFunExt`
and `#!rzk WeakFunExt`.
```

**Comment by emilyriehl** [comment]
The code looks good but the exposition could be improved. Start the new section with some text that explains what you are planning to do and how the proof will go. The citation to Rijke doesn't appear to be in the right place. If you are formalizing a specific result in that book, then you can use the title="blah" tag immediately before the definition that corresponds to that result. Here it appears you are citing an entire section, so I would do this in the body of the comment and not with a title.

Can we come up with a better name for "WeirdFunExt". Similarly how should I interpret the name of the first term you define? Feel free to ask questions about this on the zulip.

**Comment by thchatzidiamantis** [comment]
Changes to the exposition. I did not make any name changes on my own, I would be happy to discuss and rename as appropriate.

**Review by emilyriehl — APPROVED** [review]
We need a better name for WeirdFunExt, but let's fix this later.

**Pull request merged**